Lib/fontmake/instantiator.py

```diff
@@ -153,6 +153,26 @@ class InstantiatorError(Exception):
     pass
 
 
+def processRulesSwaps(rules, location, glyphNames):
```

Comment:
minor: the rest of the function names in fontmake.instantiator use snake_case, might be worth doing the same for process_rules_swaps